.github/workflows/cli.yaml

```diff
@@ -37,6 +37,12 @@ jobs:
         with:
           go-version-file: go.mod
 
+      - name: run karmadactl init use config file test
+        run: |
+          export CLUSTER_VERSION=kindest/node:${{ matrix.k8s }}
+
+          # Run custom test for workload configuration deployment
+          hack/cli-testing-init-with-config.sh
```

Comment:
Adding this CI is in the right direction, but I guess we need to add it in another pr to reduce the size of each pr,And let the CI stand alone in a separate action, rather than with the run karmadactl init test

Reply:
Okay, I will isolate it as a separate PR and place it in a distinct CI section.